src/Controls/src/Core/RefreshView/RefreshView.cs

```diff
@@ -62,7 +62,7 @@ static object OnIsRefreshingPropertyCoerced(BindableObject bindable, object valu
 		public bool IsRefreshing
 		{
 			get { return (bool)GetValue(IsRefreshingProperty); }
-			set { SetValue(IsRefreshingProperty, value); }
+			set { SetValue(IsRefreshingProperty, value, SetterSpecificity.FromHandler); }
```

Comment:
`RefreshView` isn't a Handler, so is this correct? @PureWeen does this actually need to go somewhere else?

Reply:
See Shane's comment here: https://github.com/dotnet/maui/pull/16997#discussion_r1307774329